UnityPy/helpers/TextureSwizzler.py

```diff
@@ -70,7 +70,7 @@ def swizzle(
 
 
 # this should be the amount of pixels that can fit 16 bytes
-TEXTUREFORMAT_BLOCK_SIZE_MAP: Dict[TextureFormat, Tuple[int, int]] = {
+TEXTURE_FORMAT_BLOCK_SIZE_MAP: Dict[TextureFormat, Tuple[int, int]] = {
```

Comment:
This is actually a breaking change, as the export of ``TEXTUREFORMAT_BLOCK_SIZE_MAP`` isn't restricted.
Therefore the commit containing this change has to be marked according with a !.
As the commit message of e3a398c doesn't indicate that the rename was done in it,
it would be better to move this change into a separate commit.